api/test/compartments.test.js

```diff
@@ -0,0 +1,33 @@
+import fetch from 'node-fetch';
+import { validateComponent } from './util';
+
+const NUCLEUS_INFO = {
+  id: 'nucleus',
+  letterCode: 'n',
+  name: 'Nucleus',
+  genesCount: 334,
+  metabolitesCount: 181,
+  reactionsCount: 238,
+  subsystemCount: 26,
+};
+
+describe('compartments', () => {
+  test('a compartment should have correct data', async () => {
+    const res = await fetch(
+      `${API_BASE}/compartments/nucleus?model=HumanGem&version=${HUMAN_GEM_VERSION}`
+    );
+
+    const { info, subsystems } = await res.json();
+    validateComponent(info, NUCLEUS_INFO);
+    expect(info.subsystemCount).toBe(subsystems.length);
+  });
+
+  test('a compartment should have related reactions', async () => {
+    const res = await fetch(
+      `${API_BASE}/compartments/nucleus/related-reactions?model=HumanGem&version=${HUMAN_GEM_VERSION}`
+    );
+
+    const data = await res.json();
+    expect(data.length).toBe(238);
```

Comment:
maybe use `NUCLEUS_INFO.reactionsCount` instead of 238?

Reply:
Good catch! Fixed in: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/830/commits/67eec460463e3d38bda2ae10cff7098a5220397a